M2/Macaulay2/packages/CoincidentRootLoci/documentation.m2

```diff
@@ -228,9 +228,44 @@ Inputs => {"F" => RingElement => {"a binary form ",TEX///$F\in K[x,y]$///," of d
 Outputs => {ZZ => {"the real rank of ",TEX///$F$///,", i.e., the minimum integer ",TEX///$r$///," such that there is a decomposition ",TEX///$F = c_1\,(l_1)^d+\cdots+c_r\,(l_r)^d$///," where ",TEX///$l_1,\ldots,l_r$///," are real linear forms and ",TEX///$c_1,\ldots,c_r\in\mathbb{R}$///}},
 PARA{"This method requires generally the program ",HREF{"https://www.usna.edu/CS/qepcadweb/B/QEPCAD.html","QEPCAD"}," to be installed. Source code and installation instructions for it are available at ",HREF{"https://www.usna.edu/CS/qepcadweb/INSTALL/IQ.html","Downloading and Installing QEPCAD"},"."},
 PARA{"Below we compute the real rank of a binary form of degree 7."},
-EXAMPLE {"R := QQ[x,y];","F = 2*x^7+7*x^6*y+168*x^5*y^2+140*x^4*y^3+70*x^3*y^4+21*x^2*y^5+56*x*y^6+4*y^7","realrank F"},
+PRE ///i1 : R := QQ[x,y];
```

Comment:
Why not run these examples?  That would be much better, as a way of keeping the documentation in line with what the code does, and as a way to test the code.

Reply:
Sure, but that code requires Qepcad. Now I changed it using `if CoincidentRootLoci.Options.OptionalComponentsPresent then ... else ...`